src/main/webapp/assets-lti/js/controllers/controllers.js

```diff
@@ -337,20 +337,25 @@ canvasSupportApp.controller('addUserController', ['Friend', '$scope', '$rootScop
         
         var url = '/canvasCourseManager/manager/api/v1/sections/' + sectionId + '/enrollments?enrollment[user_id]=' + $scope.friend.id + '&enrollment[enrollment_state]=active&enrollment[type]=' + thisSectionRole;
         Friend.addFriendToSection(url).then(function (resultAddFriendToSection) {
-          if (resultAddFriendToSection.data.errors) {
-            // failed to process this add
-            errors.push(sectionName);
-            $scope.addError = true;
-          } else {
-            if(resultAddFriendToSection.data.course_id) {
-              // was able to process this add
-              successes.push(sectionName);
-              if (checkedSections === sectNumber){
-                // the last request, clean up the scope
-                $scope.newUser = false;
-                $scope.none = false;
-                $scope.userAvailable  = false;
-                $scope.coursemodal.resetable = true;
+          if(resultAddFriendToSection.data.message){
+            $scope.addErrorClosed = true;
+          }
```

Comment:
Is "Can't add an enrollment to a concluded course." the only type of message we will get? We could future proof this by also checking for that in the message.